src/config.py

```diff
@@ -191,9 +191,9 @@ def bootstrap_server(self) -> List[str]:
             self.charm.model.get_relation(PEER).data[unit].get("private-address") for unit in units
         ]
         port = (
-            SECURITY_PROTOCOL_PORTS["SASL_SSL"].internal
+            SECURITY_PROTOCOL_PORTS["SASL_SSL"].external
```

Comment:
Right now it doesn't matter, but I want to clear up what we're using `INTERNAL` vs `INTERNAL` listeners for. 

`INTERNAL` listeners are addresses that we can access from *within* the Juju model (e.g a K8s namespace, or VM DNS).
`EXTERNAL` listeners are addresses that are accessible from *outside* the model. A good example would be K8s ingress/lb. 

Here, we're building `bootstrap-server` so that we can pass it to related Juju applications. If they can relate, right now, it means they're `INTERNAL`. 

In the future, we can have multi-cluster Kafka spanning clouds/models, but right now we don't have that.

Was that understood as the intention? If so, I would keep this as `INTERNAL`.